html/semantics/scripting-1/the-script-element/module/dynamic-import/microtasks/basic.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<title>Dynamic import interaction with microtask queue</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script type="module">
+import { ticker } from "./ticker.js";
+
+promise_test(async t => {
+  const getCount = ticker(1000);
+
+  try {
+    await import("<invalid>");
+    assert_unreached('import() should reject');
+  } catch (_) {}
```

Comment:
Use https://web-platform-tests.org/writing-tests/testharness-api.html#promise-rejection for these.